core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -481,12 +483,30 @@ default void setSchemaAuthorization(ConnectorSession session, String schemaName,
      * Creates a table using the specified table metadata.
      *
      * @throws TrinoException with {@code ALREADY_EXISTS} if the table already exists and {@param ignoreExisting} is not set
+     * @deprecated use {@link #createTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, SaveMode saveMode)}
      */
+    @Deprecated
```

Comment:
It is time to remove this deprecated method. Can you please follow-up?

Reply:
OK. will have a look at this.